app/institutions/dashboard/template.hbs

```diff
@@ -40,14 +40,14 @@
                     @isLoading={{this.model.taskInstance.isRunning}}
                     @title={{t 'institutions.dashboard.projects_panel'}}
             >
-                <Institutions::Dashboard::-Components::ProjectsPanel @institution={{this.institution}} />
+                <Institutions::Dashboard::-Components::ProjectsPanel @summaryMetrics={{this.summaryMetrics}} />
             </Institutions::Dashboard::-Components::Panel>
             <Institutions::Dashboard::-Components::Panel
                     local-class='departments'
                     @isLoading={{this.model.taskInstance.isRunning}}
                     @title={{t 'institutions.dashboard.departments_panel'}}
             >
-                <Institutions::Dashboard::-Components::DepartmentsPanel @institution={{this.institution}} />
+                <Institutions::Dashboard::-Components::DepartmentsPanel @departments={{this.departmentMetrics}} />
```

Comment:
Will this only load the first 10 departments? will the loadAll in the route model hook give all the institutions?